src/components/TopologySidebar/ComponentChecks.tsx

```diff
@@ -0,0 +1,116 @@
+import { useQuery } from "@tanstack/react-query";
+import { useAtom } from "jotai";
+import { useEffect } from "react";
+import { AiFillHeart } from "react-icons/ai";
+import {
+  ComponentHealthCheckView,
+  getComponentChecks
+} from "../../api/services/topology";
+import PillBadge from "../Badge/PillBadge";
+import CollapsiblePanel from "../CollapsiblePanel";
+import EmptyState from "../EmptyState";
+import { Icon } from "../Icon";
+import TextSkeletonLoader from "../SkeletonLoader/TextSkeletonLoader";
+import { refreshButtonClickedTrigger } from "../SlidingSideBar";
+import Title from "../Title/title";
+import { Status } from "../Status";
+import { Link } from "react-router-dom";
+
+type ComponentCheckLinkProps = {
+  componentCheck: ComponentHealthCheckView;
+};
+
+export function ComponentCheckLink({
+  componentCheck
+}: ComponentCheckLinkProps) {
+  return (
+    <Link
+      to={{
+        pathname: `/health`,
+        search: `?checkId=${componentCheck.id}&timeRange=1h`
+      }}
+      className={`flex flex-row justify-between w-full items-center space-x-2 p-2 rounded-md hover:bg-gray-100`}
+    >
+      <div className="flex flex-row gap-2 w-full items-center">
+        <div className="flex flex-row space-x-1 items-center flex-1">
+          <div className="text-sm flex flex-row w-full">
+            <Icon
+              className="w-4 h-auto mr-2"
+              name={componentCheck.name}
+              icon={componentCheck.check_component_relationships[0].check.icon}
+            />
```

Comment:
I don't think we need this, just the status icon (on the left) check icon and name